src/api/rest/actions/activity.js

```diff
@@ -0,0 +1,5 @@
+import { createActionSet } from 'utils';
+
+export const ACTIVITY_GET_EVENTS_RECEIVED = createActionSet(
```

Comment:
What do you think about using `redux-actions` instead of this `createActionSet` util?

Reply:
The way it's done in this PR, actions are barely seeable already : 
 - defined in [rest/actions/](https://github.com/gitpoint/git-point/pull/457/files#diff-40791c1b0969e259b980e54d84481d84)
 - mapped to paginations in [rest/reducers/paginations.js](https://github.com/gitpoint/git-point/pull/457/files#diff-0188bb8ed2ff83caf951adfd9b83aa7c)

What would be the benefit from using `redux-actions`? (I'm not quiet familiar with it)

Reply:
Here's a PR that uses `redux-actions` #269

Reply:
Redux Actions will reduce verbosity introduced by default by Redux. You can see the motivation here https://redux-actions.js.org/docs/introduction/Motivation.html

Reply:
I understand the motivation here, but against master, not this PR :/